### PR TITLE
docs(tutorial): add wrapping up page

### PR DESCRIPTION
**Pull request by mattrosno**
DO NOT MERGE this is a timed deploy.

Extends https://github.com/carbon-design-system/carbon-website/pull/1755.

### CODE REVIEW

File: src/content/tutorial/wrapping-up/wrapping-up.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+<ImageComponent cols="8" caption="Carbon t-shirt, #swag #threads #highfashion">
+
+![](/images/carbon-tee.png)
+
+</ImageComponent>
+
+## Application
+
+Complete the [application form]() to earn the digital badge and t-shirt. Once you submit the application, please allow 2-4 days for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```
Comment (kellychurchill) [inline src/content/tutorial/wrapping-up/wrapping-up.mdx]: Link to survey: https://www.surveygizmo.com/s3/5075530/Dev-Essentials-Badge-Application

File: src/content/tutorial/wrapping-up/wrapping-up.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+_\* Shirts are available for a limited time while supplies last to U.S. shipping addresses and internationally to participating FED@IBM branches._
```
Comment (kellychurchill) [inline src/content/tutorial/wrapping-up/wrapping-up.mdx]: _\* Shirts are available for a limited time while supplies last to U.S. shipping addresses and outside the U.S. to participating FED@IBM branches._

File: src/content/tutorial/wrapping-up/wrapping-up.mdx (hunk 1 above)
Comment (kellychurchill) [inline src/content/tutorial/wrapping-up/wrapping-up.mdx]: `to earn the digital badge and t-shirt` should say `to request the digital badge and t-shirt`

**Comment by mattrosno** [comment]
@jeanservaas @connor-leech can you review this page? https://deploy-preview-1794--carbon-website.netlify.com/tutorial/wrapping-up

this PR add that page and includes it in the left nav.

@kellychurchill is looking to publish it at some point next week.

File: src/content/tutorial/wrapping-up/wrapping-up.mdx (hunk 1 above)
Comment (kellychurchill) [reply src/content/tutorial/wrapping-up/wrapping-up.mdx]: I'm not seeing the link  - please add it here:

`Complete the  [application form](ttps://www.surveygizmo.com/s3/5075530/Dev-Essentials-Badge-Application) to earn` ...

File: src/content/tutorial/wrapping-up/wrapping-up.mdx [hunk 3]
```diff
@@ -25,21 +24,21 @@ Once you have completed all five steps of the tutorial, you can apply for the IB
...
 
-1. Completion of steps 1 through 5 of the React Carbon tutorial
+1. Complete steps 1 through 5 of the React Carbon tutorial
    - Step 1. [Installing Carbon](/tutorial/react-step-1)
    - Step 2. [Building pages](/tutorial/react-step-2)
    - Step 3. [Using APIs](/tutorial/react-step-3)
    - Step 4. [Creating components](/tutorial/react-step-4)
    - Step 5. [Deploying to IBM Cloud](/tutorial/react-step-5)
-2. Submission of _approved_ pull requests for steps 1 through 5 of the React Carbon tutorial in the [carbon-tutorial repository](https://github.com/carbon-design-system/carbon-tutorial).
+2. Submission of approved pull requests for steps 1 through 5 of the React Carbon tutorial in the [carbon-tutorial repository](https://github.com/carbon-design-system/carbon-tutorial).
```
Comment (kellychurchill) [inline src/content/tutorial/wrapping-up/wrapping-up.mdx]: Grammar change is not carried through here: 

 "Submission of approved" should now say "Submit approved" - in fact it should say "Submit links to approved pull requests"

**Comment by mattrosno** [comment]
> Please add the link to the form

The link was in the commit?

**Comment by kellychurchill** [comment]
> > Please add the link to the form
> 
> The link was in the commit?

The link to the form is in the review comments....

File: src/content/tutorial/wrapping-up/wrapping-up.mdx [hunk 4]
```diff
@@ -57,6 +57,6 @@ Submit your completed IBM Digital Badge application by July 30, 2019, and receiv
 
 ## Application
 
-Complete the [application form]() to earn the digital badge and t-shirt. Once you submit the application, please allow 2-4 days for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
+Complete the [application form](https://www.surveygizmo.com/s3/5075530/Dev-Essentials-Badge-Application) to request the digital badge and t-shirt. Once you submit the application, please allow 2-4 days for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```
Comment (mattrosno) [inline src/content/tutorial/wrapping-up/wrapping-up.mdx]: @kellychurchill the link is here.

Comment (kellychurchill) [reply src/content/tutorial/wrapping-up/wrapping-up.mdx]: 👍 I may have looked at the preview before it rebuilt.

**Comment by alisonjoseph** [comment]
accidentally approved while logged in to my daughters' account 😄

**Pull request merged**